package.json

```diff
@@ -30,45 +30,44 @@
   "lint-staged": {
     "*.js": [
       "prettier --write",
-      "eslint",
-      "git add"
+      "eslint"
     ]
   },
   "dependencies": {
-    "ajv": "6.12.3",
+    "ajv": "6.12.6",
     "caseless": "0.12.0",
     "clone": "2.1.2",
-    "commander": "3.0.2",
+    "commander": "7.1.0",
     "content-type": "1.0.4",
     "curl-trace-parser": "0.0.10",
-    "deep-equal": "1.1.1",
+    "deep-equal": "2.0.5",
     "http-string-parser": "0.0.6",
-    "json-pointer": "0.6.0",
+    "json-pointer": "0.6.1",
     "media-typer": "1.1.0",
     "tv4": "1.3.0",
-    "url": "0.11.0"
+    "url-parse": "1.5.1"
```

Comment:
It'd be great if you could have a look at #536. Technically `url` is just a dev-dependency right now.

Reply:
Based on the code of validate.js - there's still the `validateURI` function using deep-clone to do comparison of a query-string regardless of ordering of `key=value` pairs.

Reply:
building CommonJS sure - I might look into it.
You're right that `url` could be a devDependency, as NodeJs has its version as part of core.
Have to 🤔 about it a bit more.

Reply:
@realityking I'm more or less thinking about actually reusing `uri-js` instead of any url-parse/url/native-url or what-not (that package is anyway used by `ajv`, so it is present at all cost)

Reply:
Not a bad idea. Another idea could be to move to the native URL API (https://nodejs.org/api/url.html#url_the_whatwg_url_api) and either polyfill for older browsers or ask users to polyfill themselves.